bin/publish.ts

```diff
@@ -49,8 +49,8 @@ const EVENT_TO_LEVEL: Record<EventType, LogLevel> = {
   upload: 'verbose',
 };
 
-class ConsoleProgress implements IPublishProgressListener {
-  public onPublishEvent(type: EventType, event: IPublishProgress): void {
-    log(EVENT_TO_LEVEL[type], `[${event.percentComplete}%] ${type}: ${event.message}`);
+export class ConsoleProgress implements IPublishProgressListener {
+  public onPublishEvent(type: EventType, event: IPublishProgress, forceStdOut?: boolean): void {
```

Comment:
Bit of a chicken and egg here but we need to do this to force shell logging messages to be printed to stdout in the CLI as they were being printed to stdout in cdk-assets directly and stderr apparently breaks some CI enviornments

Reply:
see: https://github.com/aws/aws-cdk/pull/32824

Reply:
Don't worry about logging to stderr breaking CI environments.

If that was a problem for this tool, it would already have been a problem before and we would have either fixed it already or at least have an open ticket about it. I don't think we do.

That is not what you're here for, don't get distracted. Maintaining current behavior is the only thing you need to do.